client/l2tp_client.c

```diff
@@ -1268,24 +1353,20 @@ int main(int argc, char **argv)
 
       if (ready_cnt == broker_cnt || is_timeout(&timer_collect, 20))
         break;
-    }
 
-    // Select the first available broker and use it to establish a tunnel
-    main_context = NULL;
-    for (i = 0; i < broker_cnt; i++) {
-      if (brokers[i].ctx->standby_available) {
-        brokers[i].ctx->standby_only = 0;
-        main_context = brokers[i].ctx;
+      /* first available broker just use the first one available */
```

Comment:
Can we use consistent comments (in this case `//`)? And start them with a capital letter.